dev-util/geany-plugins/geany-plugins-1.34.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 vala
+
+DESCRIPTION="A collection of different plugins for Geany"
+HOMEPAGE="https://plugins.geany.org"
+SRC_URI="https://plugins.geany.org/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="+gtk3 ctags debugger enchant git gpg gtkspell lua markdown multiterm nls pretty-printer python scope soup"
+REQUIRED_USE="
+	gtk3? ( !debugger !multiterm !python )
+	!gtk3? ( !markdown )
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+COMMON_DEPEND="
+	>=dev-util/geany-1.34[gtk3=]
+	dev-libs/glib:2
+	gtk3? ( x11-libs/gtk+:3 )
+	!gtk3? ( x11-libs/gtk+:2 )
+	ctags? ( dev-util/ctags )
+	debugger? ( x11-libs/vte:0 )
+	enchant? ( app-text/enchant )
+	git? ( dev-libs/libgit2:= )
+	gpg? ( app-crypt/gpgme:1= )
+	gtkspell? (
+		gtk3? ( app-text/gtkspell:3= )
+		!gtk3? ( app-text/gtkspell:2 )
+		)
+	lua? ( dev-lang/lua:0= )
+	markdown? (
+		app-text/discount
+		net-libs/webkit-gtk
+		)
+	multiterm? (
+		$(vala_depend)
+		>=x11-libs/vte-0.28:0
+		)
+	pretty-printer? ( dev-libs/libxml2:2 )
+	python? (
+		dev-python/pygtk[${PYTHON_USEDEP}]
+		${PYTHON_DEPS}
+		)
+	scope? (
+		gtk3? ( x11-libs/vte:2.91 )
+		!gtk3? ( x11-libs/vte:0 )
+		)
+	soup? ( net-libs/libsoup:2.4 )
+"
+RDEPEND="${COMMON_DEPEND}
+	scope? ( sys-devel/gdb )
+"
+DEPEND="${COMMON_DEPEND}
+	nls? ( sys-devel/gettext )
+	virtual/pkgconfig
```

Comment:
(Note that if EAPI-7 works, this goes into BDEPEND)

Reply:
Moved to BDEPEND.